symplyphysics/laws/thermodynamics/chemical_potential_is_particle_count_derivative_of_free_energy.py

```diff
@@ -0,0 +1,71 @@
+from sympy import Eq, Derivative, Point2D
+from symplyphysics import (
+    units,
+    dimensionless,
+    Quantity,
+    Symbol,
+    Function,
+    print_expression,
+    validate_input,
+    validate_output,
+)
+from symplyphysics.core.geometry.line import two_point_function
+
+# Description
+## The chemical potential of the system is the amount of energy the system absorbs or releases
+## due to the introduction of a particle into the system, i.e. when the particle count increases
+## by one.
+
+# Law: mu = (dF/dN)_(T, V)
+## mu - chemical potential
+## F = F(T, V, N) - [Helmholtz free energy](./helmholtz_free_energy_via_internal_energy.py)
+## N - particle count
+## T - temperature
+## V - volume
+## (d/dN)_(T, V) - derivative w.r.t. particle count at constant temperature and volume
+
+chemical_potential = Symbol("chemical_potential", units.energy)
+free_energy = Function("free_energy", units.energy)
+particle_count = Symbol("particle_count", dimensionless)
+temperature = Symbol("temperature", units.temperature)
+volume = Symbol("volume", units.volume)
+
+# Note that `temperature` and `volume` are only used to show that the free_energy
+# function depends on them and they are held constant during differentiation with
+# respect to `particle_count`.
```

Comment:
Is it important to know that free_energy function depends on them? I doubt it. And removing irrelevant parameters will make this law cleaner.